alma/alma-monthlypayments-magento2
# Add rest in send collect and change trigger

### Reason for change



[[Linear task](https://linear.app/almapay/issue/ECOM-xxx/)
](https://linear.app/almapay/issue/ECOM-2247/[ac]-add-rest-in-send-collect)
### Code changes

Add rest to url and change trigger from after save to loadconfig



### How to test

With a new instance, save an api key and check metabase 

_As a reviewer, you are encouraged to test the PR locally._



### Checklist for authors and reviewers




### Non applicable
